frontend/src/test/java/olumofe/raphael/frontend/CarControllerTest.java

```diff
@@ -29,6 +28,8 @@ public void checkObjectIsCreated() throws Exception {
 
         ResponseEntity<String> response = restTemplate.getForEntity(
                 new URL("http://localhost:" + port + "/cars").toString(), String.class);
+        // assertEquals("Hello Controller", response.getBody());
         assertNotNull(response.getBody());
```

Comment:
This is not a useful assertion. We should check if the input we give produces the output we expect. This is testing whether we get something at all (which is sometimes useful).